joint-discovery-gatk4-local.wdl

```diff
@@ -102,8 +101,7 @@ workflow JointGenotyping {
   call DynamicallyCombineIntervals {
     input:
       intervals = unpadded_intervals_file,
-      merge_count = merge_count,
-      preemptible = preemptible
```

Comment:
Why did you hard code the preemptible attempts?

Reply:
I want to keep the variables as consistent as possible with the parent wdl, it makes updating the wdl easier as there will be less differences to consider. It would be great if the parent wdl in dsde-pipelines were not hardset and followed the wdl template .
https://docs.google.com/document/d/1uiapGNu4lWTmiV_xHjvDfHuDMYxTpG229gqNutQQhfQ/edit#heading=h.9kzq7s20lglw